### PR TITLE
RabbitMQ source kamelet

### LINKED ISSUE #44: Add RabbitMQ source Kamelet

### PR DESCRIPTION
**Pull request by oscerd**
Fixes #44

### CODE REVIEW

File: rabbitmq-source.kamelet.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  flow:
+    from:
+      uri: "rabbitmq://{{exchangeName}}"
+      parameters:
+        password: "{{password}}"
+        username: "{{username}}"
+        addresses: "{{addresses}}"
+      steps:
+      - convert-body-to:
+          type: "java.lang.String"
```
Comment (nicolaferraro) [inline rabbitmq-source.kamelet.yaml:53]: Does it need this conversion? I guess it already produces binary data that can be sent over the wire, right?

Comment (oscerd) [reply rabbitmq-source.kamelet.yaml:53]: Yes, bytes only

Comment (nicolaferraro) [reply rabbitmq-source.kamelet.yaml:53]: Yeah, so I would send it directly to the sink

Comment (oscerd) [reply rabbitmq-source.kamelet.yaml]: Fixed

**Pull request merged**